### PR TITLE
Update aws extension

### PR DESCRIPTION
**Pull request by vineus**
## Description

Enhanced the S3 bucket command: navigate buckets by folders

## Screencast

https://github.com/raycast/extensions/assets/467126/3ff6d02d-2e13-420b-a60a-25ddf000682f

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Review by JonathanWbn — APPROVED** [review]
Looks great, nice improvement 💪

File: extensions/amazon-aws/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Amazon AWS Changelog
 
+## [S3 folders] - 2024-04-14
+
+- Added support for navigating S3 buckets using folders folders
```
Comment (Hodglim) [inline extensions/amazon-aws/CHANGELOG.md:5]:
```suggestion
- Added support for navigating S3 buckets using folders
```

**Comment by Hodglim** [comment]
Looks like an awesome update, very nice! 🙌🏼

File: extensions/amazon-aws/CHANGELOG.md (hunk 1 above)
Comment (vineus) [reply extensions/amazon-aws/CHANGELOG.md:5]: ooops 😅

**Review by Hodglim — APPROVED** [review]

**Review by pernielsentikaer — APPROVED** [review]

**Pull request merged**